doc/man3/OSSL_PARAM_TYPE.pod

```diff
@@ -230,6 +266,47 @@ OSSL_PARAM_set_octet_ptr() sets the OCTET string pointer in the parameter
 referenced by B<p> to the values B<val>.
 The length of the OCTET string is provided by B<used_len>.
 
+OSSL_PARAM_locate_get_TYPE() behaves exactly like:
```

Comment:
What does 'behaves exactly like' mean?

Reply:
Suggestions for alternative wordage are welcome.

Reply:
Is _is isomorphic to_ any better?